packages/node/src/stellar/block.stellar.ts

```diff
@@ -118,7 +118,7 @@ export class StellarBlockWrapped implements StellarBlockWrapper {
     filter: SorobanEventFilter,
     address?: string,
   ): boolean {
-    if (address && !stringNormalizedEq(address, event.contractId.toString())) {
+    if (address && !stringNormalizedEq(address, event.contractId?.toString())) {
```

Comment:
Does `stringNormalizedEq` accept undefined as the second param?

Reply:
https://github.com/subquery/subql-stellar/blob/33c6861716ba5f8b5c864dabc4a082d49ac1b0ee/packages/node/src/utils/string.ts#L5

Looks it does from current main